### PR TITLE
Tried fixing the issue #959 Transformation idea: vector access like .…

### PR DESCRIPTION
**Pull request by KishkinJ10**
…x to [o] & also made some updates in the read me file , please take a look at  these changes as i am a beginner , it will help me very much in contributions.

### CODE REVIEW

File: common/src/main/java/com/graphicsfuzz/common/util/IRandom.java [hunk 1]
```diff
@@ -1,49 +0,0 @@
-/*
- * Copyright 2018 The GraphicsFuzz Project Authors
- *
```
Comment (afd) [inline common/src/main/java/com/graphicsfuzz/common/util/IRandom.java:3]: Why have you deleted this file?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # How to Contribute
 
-We'd love to accept your patches and contributions to this project. There are
+We'd love to accept your patches ,bug and issue fixes and contributions to this project. There are
```
Comment (afd) [inline CONTRIBUTING.md:3]: This change has nothing to do with the issue you are addressing. (And also it is typographically incorrect.)

File: generator/src/main/java/com/graphicsfuzz/generator/fuzzer/OpaqueExpressionGenerator.java [hunk 3]
```diff
@@ -1076,6 +1077,33 @@ public Expr apply(Expr expr, BasicType type, boolean constContext, int depth,
...
+    private IdentityVectorComponent(){
+      super(Arrays.asList(BasicType.BOOL), false);
+    }
+
+    @Override
+    public Expr apply(Expr expr, BasicType type, boolean constContext, int depth,
+                      Fuzzer fuzzer){
+
+      assert type == BasicType.BOOL;
+      result
```
Comment (afd) [inline generator/src/main/java/com/graphicsfuzz/generator/fuzzer/OpaqueExpressionGenerator.java:1090]: This doesn't look like it compiles.

**Review by afd — COMMENTED** [review]
This PR has major problems and does not compile.

**Comment by KishkinJ10** [comment]
Sorry sir , I am a beginner and I wanted to start my open source journey by
making a contribution

**Pull request closed without merging**